hadoop-ozone/cli-admin/src/main/java/org/apache/hadoop/hdds/scm/cli/datanode/UsageInfoSubcommand.java

```diff
@@ -147,9 +147,12 @@ private void printInfo(DatanodeUsage info) {
     System.out.printf("%-24s: %s (%s) %n", "Remaining Allocatable",
         (info.getRemaining() - info.getCommitted()) + " B",
         StringUtils.byteDesc((info.getRemaining() - info.getCommitted())));
-    System.out.printf("%-24s: %s (%s) %n%n", "Free Space To Spare",
+    System.out.printf("%-24s: %s (%s) %n", "Free Space To Spare",
         info.getFreeSpaceToSpare() + " B",
         StringUtils.byteDesc(info.getFreeSpaceToSpare()));
+    System.out.printf("%-13s: %s (%s) %n%n", "Reserved",
+        info.getReserved() + " B", 
+        StringUtils.byteDesc(info.getReserved()));
```

Comment:
Please move the extra `%n` as a `System.out.println()` at the end.  This will reduce future change if adding another item (like here `Free Space To Spare` is being changed).